mypy/fastparse.py

```diff
@@ -979,11 +1003,23 @@ def make_argument(self, arg: ast3.arg, default: Optional[ast3.expr], kind: ArgKi
             arg_type = None
             if annotation is not None:
                 arg_type = TypeConverter(self.errors, line=arg.lineno).visit(annotation)
+            elif isinstance(default, ast3.Constant):  # #12352.
```

Comment:
We should probably do this only *after* trying to find the type in a type comment.